### PR TITLE
Added EPOS4 recipe

### PR DESCRIPTION
**Pull request by jackal1-66**
The build was tested in a Ubuntu machine

### CODE REVIEW

File: epos4.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+requires:
+  - ROOT
+  - fastjet
+env:
+  EPOVSN: "4.0.0"
+  EPO: "${EPOS_ROOT}/epos4/"
+  LIBDIR: "${EPO}bin"
+  OPT: "./"
+  HTO: "./"
+  CHK: "./"
```
Comment (ktf) [inline epos4.sh:14]: These will be set at build time for the dependencies of EPOS4. Is that what you really need?

File: epos4.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+set EPOS_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv EPOS_ROOT \$EPOS_ROOT
+setenv EPOVSN 4.0.0
+# Final slash is required by EPOS, please leave it be
+setenv EPO \$::env(EPOS_ROOT)/epos4/
+prepend-path PATH \$::env(EPO)bin
+setenv LIBDIR \$::env(EPO)bin
+setenv OPT ./
+setenv HTO ./
+setenv CHK ./
```
Comment (ktf) [inline epos4.sh:76]: Is this what you really need? These will be set at runtime when you load the module.

Comment (jackal1-66) [reply epos4.sh:76]: It's in the tutorial also of the installation of EPOS4 standalone. They need to be set and that should be the default one, but the user can change it afterwards obviously.

File: epos4.sh (hunk 1 above)
Comment (jackal1-66) [reply epos4.sh]: I removed them cause they aren't needed (for now).

**Review by maireiphc — APPROVED** [review]

**Pull request merged**